sample-config.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: "networkoperator.openshift.io/v1"
+kind: "NetworkConfig"
+metadata:
+  name: "default"
+  namespace: cluster-network-operator
+spec:
+  serviceNetwork: "172.30.0.0/16"
+  clusterNetworks:
+    - cidr: "10.128.0.0/14"
+      hostSubnetLength: 9
```

Comment:
Should we test a multiple cidr case?

Reply:
we do, in the unit tests. These are the openshift-ansible defaults, so I'd like to keep them.